dropwizard-logging/src/test/java/io/dropwizard/logging/DefaultLoggingFactoryTest.java

```diff
@@ -71,6 +71,7 @@ public void canParseNewLoggerFormat() throws Exception {
         assertThat(fileAppenderFactory.getCurrentLogFilename()).isEqualTo("${new_app}.log");
         assertThat(fileAppenderFactory.getArchivedLogFilenamePattern()).isEqualTo("${new_app}-%d.log.gz");
         assertThat(fileAppenderFactory.getArchivedFileCount()).isEqualTo(5);
+        assertThat(fileAppenderFactory.getFilterFactories()).containsExactly(new TestFilterFactory(), new SecondTestFilterFactory());
```

Comment:
After another look, I beleive we could even use `ImmutableList<FilterFactory<ILoggingEvent>>` if we generify `appenderFactory` and `fileAppenderFactory`.